config/environments/production.rb

```diff
@@ -59,15 +59,21 @@
   # Precompile additional assets.
   # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
   # config.assets.precompile += %w( search.js )
+
   config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
   config.action_mailer.smtp_settings = {
-    :address => "smtp.gmail.com",
-    :port    => 587,
-    :user_name => ENV['gmail_username'],
-    :password => ENV['gmail_password'],
-    :authentication => "plain",
+    :user_name => ENV['SENDGRID_USERNAME'],
+    :password => ENV['SENDGRID_PASSWORD'],
```

Comment:
Sendgrid now recommends using an API key instead of u/p credentials; could we use that strategy instead?